spec/system/potential_user_browses_recipes_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.feature "A potential user browses recipes created by others" do
+  scenario "by default, when visiting the homepage" do
+    user = create(:user)
+    recipes = create_list(:recipe, 5, user: user)
+
+    visit root_path
+
+    expect(page).to have_content(recipes[0].id)
+    expect(page).to have_content(recipes[1].id)
+    expect(page).to have_content(recipes[2].id)
+    expect(page).to have_content(recipes[3].id)
+    expect(page).to have_content(recipes[4].id)
```

Comment:
What do you think about just looping through these?

```ruby
recipes.each do |recipe|
  expect(page).to have_content(recipe.title)
end
```

I'd also consider checking for the recipe title rather than the id. Feature tests should ideally check for the content the user is supposed to see.

Reply:
Ah yes! I knew there was a better way to do this. Thanks!